controllers/helper/userHelper.js

```diff
@@ -0,0 +1,328 @@
+import User from "../../models/userSchema.js";
+import {
+  getUserValidationErrors,
+  emailRegex,
+  passwordRegex,
+} from "../../utils.js";
+
+export const create = async (userData) => {
+  try {
+    //For third party user registering
+    if ("third_party_user_id" in userData || "third_party_type" in userData) {
```

Comment:
"third_party_user_id" in userData || "third_party_type" in userData && typeof userData.third_party_user_id !== "number"
one if statement will reduced by this